src/test-helpers.tsx

```diff
@@ -112,6 +112,22 @@ export const fakeUser: ExternalUser = Object.freeze({
   username: 'user-1234',
 });
 
+export const fakeExternalLinterMessage = Object.freeze({
+  column: 2,
+  context: ['<code>'],
+  description: 'To prevent vulnerabilities...',
+  file: 'chrome/content/youtune.js',
+  for_appversions: {
+    '{ec8030f7-c20a-464f-9b0e-13a3a9e97384}': ['4.0b1'],
+  },
+  id: [],
+  line: 226,
+  message: 'on* attribute being set using setAttribute',
+  tier: 3,
+  type: 'notice',
+  uid: '9a07163bb74e476c96a2bd467a2bbe52',
+}) as ExternalLinterMessage;
```

Comment:
Could you type the variable instead?

```
export const fakeExternalLinterMessage: ExternalLinterMessage = ...
```

TS might complain that `type` has the wrong value (because `'"notice"' !== string`) but maybe not.

Reply:
> TS might complain that type has the wrong value (because `'"notice"' !== string`)

It does, yeah. That's why I used the type assertion. I considered adding a comment but didn't think it was so useful since TS will complain if you try to change it anyway.